src/controller/account.js

```diff
@@ -16,4 +17,28 @@ account.get('/user', jwtVerifier({ secret }), async (req, res) => {
   }
 });
 
+account.post('/topup', jwtVerifier({ secret }), async (req, res) => {
+  try {
+    const { id } = req.user;
+    const { topUp } = req.body;
```

Comment:
pls call this amount instead and we should validate is also (it cannot be under 0)